CHANGELOG.md

```diff
@@ -3,6 +3,117 @@
 All notable changes to this project will be documented in this file.
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
+## 6.0.0-alpha.11
+
+_Dec 8, 2022_
+
+We'd like to offer a big thanks to the 7 contributors who made this release possible. Here are some highlights ✨:
+
+- 🚀 Add dragging support for `DateRangePicker` (#6763) @LukasTy
+- ✨ Add missing lazy-loading feature to `DataGridPremium` (#7124) @m4theushw
```

Comment:
Not knowing what this PR did, I did not understand the highlight.
I thought you developed a new feature that the community thought was missing.
But you ""just"" added a feature already developed and that we forgot to call on the premium component.

Maybe we could phrase it better

Reply:
Maybe this would be more clear?

```suggestion
- ✨ Fix lazy-loading feature not working in `DataGridPremium` (#7124) @m4theushw
```